src/lib/swapper/swappers/ThorchainSwapper/endpoints.ts

```diff
@@ -67,22 +67,22 @@ export const thorchainApi: Swapper2Api = {
   },
 
   checkTradeStatus: async ({
-    txId,
-  }): Promise<{ status: TxStatus; buyTxId: string | undefined; message: string | undefined }> => {
+    txHash,
+  }): Promise<{ status: TxStatus; buyTxHash: string | undefined; message: string | undefined }> => {
     try {
       // thorchain swapper uses txId to get tx status (not trade ID)
-      const { buyTxId } = await getTradeTxs({ tradeId: txId })
-      const status = buyTxId ? TxStatus.Confirmed : TxStatus.Pending
+      const { buyTxId: buyTxHash } = await getTradeTxs({ tradeId: txHash })
```

Comment:
Since we're effectively lying here and `buyTxId` is either an actual Txid (with `0x` or a hash, might as well have `getTradeTxs` return `buyTxHash` vernacular directly? Doesn't need to be tackled in this PR as this would be a huge type lift

Reply:
resolved in next PR as per https://github.com/shapeshift/web/pull/4884#discussion_r1252251787 🙂

Reply:
yes ser, I'm leaning towards going back to `txId` vernacular where the identifier may be a tx hash, or an actual tx ID (like in cowswap). The identifier is only used to identify the transaction to the swapper, so merging `hash | id` into just `id` makes sense